lib/src/kbaserelationengine/search/ElasticIndexingStorage.java

```diff
@@ -1306,6 +1452,10 @@ private void createAccessTable(String indexName, Map<String, Object> mappings) t
         props.put("groups", new LinkedHashMap<String, Object>() {{
             put("type", "integer");
         }});
+        // List of external workspaces containing DataPalette pointing to this object
+        props.put("extpub", new LinkedHashMap<String, Object>() {{
+            put("type", "integer");
+        }});
```

Comment:
is the -1 flag still relevant?

Reply:
I'll remove -1 in separate PR. It's not used and doesn't do any harm anyway.

Reply:
ok